i18n/translation-en.json

```diff
@@ -213,5 +215,12 @@
     "pan": "Pan",
     "select": "Select",
     "Marquee": "Marquee Zoom"
+  },
+  "watermark": {
```

Comment:
Can you move this inside the `option` object please.

Reply:
i have moved it in